src/shell/dotnet/Shell/Fdc3/ResolverUi/Pages/SimpleResolverUiPage.xaml

```diff
@@ -0,0 +1,203 @@
+                                    <RowDefinition Height="*"/>
+                                </Grid.RowDefinitions>
+
+                                <ItemsPresenter Grid.Row="0"/>
+                                
+                                <Button 
+                                    x:Name="OpenAppButton" 
+                                    Click="OpenApp_Click" 
+                                    Content="Open New Instance"
+                                    Grid.Row="1"
+                                    CommandParameter="{Binding Name}"
+                                    Style="{StaticResource ButtonStyle}"
+                                    Width="180"/>
+                                
+                            </Grid>
+                        </Expander>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+
+        <Style TargetType="DataGridRow">
+            <Setter Property="Background" Value="Transparent"/>
+            <Setter Property="BorderBrush" Value="Transparent"/>
+            <Setter Property="Visibility" Value="{Binding Path=., Converter={StaticResource SimpleResolverUIRowVisibilityConverter}, ConverterParameter=1}"/>
+            <Style.Triggers>
+                <Trigger Property="IsSelected" Value="True">
+                    <Setter Property="Background" Value="#FF177B39"/>
+                </Trigger>
+            </Style.Triggers>
+        </Style>
+
+        <Style TargetType="DataGrid">
+            <Setter Property="Background" Value="Transparent"/>
+            <Setter Property="BorderBrush" Value="Transparent"/>
+            <Setter Property="RowBackground" Value="Transparent"/>
+            <Setter Property="ColumnHeaderStyle">
+                <Setter.Value>
+                    <Style TargetType="DataGridColumnHeader">
+                        <Setter Property="Background" Value="Transparent"/>
+                        <Setter Property="BorderBrush" Value="Transparent"/>
+                        <Setter Property="VerticalAlignment" Value="Center" />
+                        <Setter Property="HorizontalAlignment" Value="Center" />
+                    </Style>
+                </Setter.Value>
+            </Setter>
+
+            <Setter Property="CellStyle">
+                <Setter.Value>
+                    <Style TargetType="DataGridCell">
+                        <Setter Property="Background" Value="Transparent"/>
+                        <Setter Property="FontWeight" Value="SemiBold"/>
+                        <Setter Property="BorderBrush" Value="Transparent"/>
+                    </Style>
+                </Setter.Value>
+            </Setter>
+        </Style>
+    </Page.Resources>
+
+    <Border
+        x:Name="ResolverUiBorder"
+        Background="Transparent">
+        
+        <Grid
+            Background="Transparent">
+            
+            <DataGrid 
```

Comment:
I know styling/UX is a bit out of scope here but for the simple view I think a simple ListBox with some DataTemplate would suffice.

Reply:
I think there was an ask to have the app ids and under them the instances available. and we should have a button indicating there is possibility to open a new instance.